tests/Application/config/packages/setono_sylius_image.yaml

```diff
@@ -7,3 +7,5 @@ setono_sylius_image:
         sylius_shop_product_small_thumbnail: ~
         sylius_shop_product_thumbnail: ~
         sylius_shop_product_large_thumbnail: ~
+    processable_resources: 
```

Comment:
Should we remove the `filter_sets` option above?

Reply:
Right now you use the `filter_sets` as the 'all' variants option which may be a good idea, but in that case we should maybe rename it to `default_variants`?

Reply:
Right now you use the `filter_sets` as the 'all' variants option which may be a good idea, but in that case we should maybe rename it to `default_variants`?